hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/om/PrepareSubCommand.java

```diff
@@ -55,42 +55,47 @@ public class PrepareSubCommand implements Callable<Void> {
   private OMAdmin parent;
 
   @CommandLine.Option(
-      names = {"-id", "--service-id"},
-      description = "Ozone Manager Service ID",
+      names = {"-id", "--id", "--service-id"},
+      description = "Ozone Manager Service ID." + "Command line option " +
+          "-id will be removed in later versions.",
       required = true
   )
   private String omServiceId;
 
   @CommandLine.Option(
-      names = {"-tawt", "--transaction-apply-wait-timeout"},
+      names = {"-tawt", "--tawt", "--transaction-apply-wait-timeout"},
```

Comment:
Please don't add `--tawt` or other similar abbreviated long options.  Options should be short or long, but not a mix.  How are users supposed to remember `tawt` and other acronyms?